client/http/client_test.go

```diff
@@ -0,0 +1,220 @@
+package http_test
+
+import (
+	"encoding/json"
+	httpkaki "github.com/env0/terraform-provider-env0/client/http"
```

Comment:
tf?

Reply:
Woops, changed in https://env0.slack.com/archives/C019V93M48P/p1622556211046200